client/client-multi/client-c/src/kaa_common.h

```diff
@@ -66,26 +66,35 @@ typedef enum {
     KAA_SERVICE_LOGGING = 4,
 } kaa_service_t;
 
+/**
+ * @brief Unique identifier used to represent client transport channel implementations.
+ */
+typedef struct {
+    uint32_t id;
+    uint16_t version;
+} kaa_transport_protocol_info_t;
```

Comment:
I agree, both documentation and struct name don't match to each other. This structure should represent unique identifier for a transport protocol.